node/src/service.rs

```diff
@@ -100,6 +99,8 @@ impl ParachainRuntimeApiImpl for amplitude_runtime::RuntimeApiImpl<Block, Amplit
 impl ParachainRuntimeApiImpl for pendulum_runtime::RuntimeApiImpl<Block, PendulumClient> {}
 impl ParachainRuntimeApiImpl for foucoco_runtime::RuntimeApiImpl<Block, FoucocoClient> {}
 
+const LOG_TARGET_SYNC: &str = "sync::cumulus";
```

Comment:
Ah just one thing. Let's maybe remove this constant and use `"sync::cumulus"` in the places where the constant is mentioned. To me, having this constant gives more significance to this log target level although it's actually not really important. And for the other log with target `"cumulus-network"` we also didn't define a constant so let's just remove this. I know it was like this in the template.